comicScraper.py

```diff
@@ -180,10 +210,23 @@ def main():
 
     # set variables from arguments
     startURL = arguments.URL
+    singleIssue = False
+    if "?id=" in startURL:
+        singleIssue = True
+        print("You have provided the link for a single issue.")
+        print("When providing the URL for the info page, this script will download all issues in the series.")
+
     if arguments.folder != None:
+        print(f"Title detected, using {arguments.folder} as title")
         comicTitle = arguments.folder
     else:
-        comicTitle = getComicTitle(startURL)
+        print("No title specified. Reading title from Url...")
+        comicTitle = getComicTitle(startURL, singleIssue)
+        print("Using title: " + comicTitle)
+
+    if arguments.complete != None and singleIssue == False:
```

Comment:
this can be switched to `arguments.complete == True` if we switch up the input parameter a few comments up

Reply:
Updated